neptune/codelist.go

```diff
@@ -210,3 +220,23 @@ func (n *NeptuneDB) GetCode(ctx context.Context, codeListID, edition string, cod
 		Code: code,
 	}, nil
 }
+
+// convert a flat array of record values into  a 2d array of records
+func createRecords(values []string, stride int) ([][]string, error) {
+	var records = [][]string{}
+	if len(values)%stride != 0 {
+		return nil, errors.New(fmt.Sprintf("List length is not divisible by %d", stride))
```

Comment:
You can use ` fmt.Errorf("List length is not divisible by %d", stride)` to do the same 😎